backend/src/main/java/com/bakdata/conquery/models/query/queryplan/filter/FilterNode.java

```diff
@@ -8,11 +8,10 @@
 
 @AllArgsConstructor
 @ToString
-public abstract class FilterNode<FILTER_VALUE> extends EventIterating {
+public abstract sealed class FilterNode<FILTER_VALUE> extends EventIterating permits EventFilterNode, AggregationResultFilterNode {
```

Comment:
benutzen wir schon irgendwo java 17 (Preview) switch with pattern matching oder warum sealing 👀

Reply:
ne, aber ich hab die harte annahme, dass es nur EventFilter oder AggregationResultFilter gibt, in den Code gesteckt. Deswegen muss ich die Klasse sealen.